dashboard/test/controllers/activities_controller_test.rb

```diff
@@ -1223,4 +1223,60 @@ def new_level
     # panda_panda contains a panda emoji, ensure that it's gone
     assert_equal user_level.level_source.data, 'Panda'
   end
+
+  test "milestone updates activities for multi assessments" do
+    Gatekeeper.set('activities', value: false)
+
+    post :milestone, params: @milestone_params
+    assert_nil Activity.find_by(user_id: @user, level_id: @script_level.id, script_id: @script_level.script.id)
```

Comment:
The only changes to these tests, compared with how they appeared in https://github.com/code-dot-org/code-dot-org/pull/29263, are:
* set gatekeeper flag to disable non-multi-assessment activity writes
* use Activities instead of now-abandoned AssessmentActivities